packages/ag-charts-enterprise/src/series/treemap/treeMapLabelFormatter.test.ts

```diff
@@ -0,0 +1,218 @@
+import { _Scene } from 'ag-charts-community';
+
+import {
+    formatSingleLabel,
+    formatStackedLabels,
+    generateLabelSecondaryLabelFontSizeCandidates,
+    maximumValueSatisfying,
+} from './treeMapLabelFormatter';
+
+const { Text } = _Scene;
+
+const wrap = jest.fn();
+const computeBBox = jest.fn();
+
+Text.wrap = wrap;
+Text.prototype.computeBBox = computeBBox;
+
+afterEach(() => {
+    jest.restoreAllMocks();
+});
+
+describe('generateLabelSecondaryLabelFontSizeCandidates', () => {
```

Comment:
Nit: Could you add an enclosing `describe` block for the scope of the entire module under test?